SwiftRustIntegrationTestRunner/SwiftRustIntegrationTestRunnerTests/OptionTests.swift

```diff
@@ -97,6 +97,19 @@ class OptionTests: XCTestCase {
         
         XCTAssertNil(rust_reflect_option_opaque_rust_type(nil))
     }
+
+    func testSwiftCallRustWithOptionRefOpaqueRustType() throws {
```

Comment:
Mind plopping this on here:

```rust
/// Verify that we can pass and receive an `Option<&RustType>`.
///
/// We deinitialize the first reference and create a second to confirm that
/// deinitializing the reference does not deinitialize the Rust type.
```

We'll eventually document every test in this way, so I want to enforce it for new ones.

The idea is that a new contributor can read the test docs and have a good sense of whether or not the test relates to what they're trying to do.

I was going to just add it myself but then I saw the `.rs.rustfmt` file so while commenting on that I figured I'd ask for this as well.

Reply:
Good idea, done!